plugins/enhanced_titles/__init__.py

```diff
@@ -0,0 +1,429 @@
+    "fra" : ["le", "la", "les", "un", "une", "des", "l'"],
+    "deu" : ["der", "den", "die", "das", "dem", "des", "den"],
+    "por" : ["o", "os", "a", "as", "um", "uns", "uma", "umas"]
+}
+_articles[""] = [value for values in _articles.values() for value in values]
+
+# Prepositions and conjunctions with 3 letters or less.
+_other_minor_words = {
+    "eng" : ["so", "yet", "as", "at", "by", "for", "in", "of", "off", "on", "per",
+            "to", "up", "via", "and", "as", "but", "for", "if", "nor", "or"],
+    "spa" : ["mas", "que", "en", "con", "por", "de", "y", "e", "o", "u", "si", "ni"],
+    "ita" : ["di", "a", "da", "in", "con", "su", "per", "tra", "fra", "e", "o", "ma", "se"],
+    "fra" : ["à", "de", "en", "par", "sur", "et", "ou", "que", "si"],
+    "deu" : ["bis", "für", "um", "an", "auf", "in", "vor", "aus", "bei", "mit",
+            "von", "zu", "la", "so", "daß", "als", "ob", "ehe"],
+    "por" : ["dem", "em", "por", "ao", "à", "aos", "às", "do", "da", "dos", "das",
+            "no", "na", "nos", "nas", "num", "dum", "e", "mas", "até", "em", "ou",
+            "que", "se", "por"]
+}
+_other_minor_words[""] = [value for values in _other_minor_words.values() for value in values]
+
+
+class ReleaseGroupHelper(MBAPIHelper):
+    """API Helper to retreive release group information.
+    """
+
+    def get_release_group_by_id(self, release_id, handler, inc = None):
+        """Gets the information for a release group.
+        """
+        return self._get_by_id("release-group", release_id, handler, inc)
+
+
+class SortTagger:
+    """Sets the titlesort and albumsort tags.
+
+    First, it checks if there is already a sort name available in one of the
+    aliases. If it does not find any, it swaps the prefix if there is one in
+    the title.
+    """
+
+    def _select_alias(self, aliases, name):
+        """Selects the first alias where the names match and the sort name is
+        different.
+
+        Args:
+            aliases (list): One dictionary for each alias available.
+            name (str): Title of the album/track.
+
+        Returns:
+            (str): The sort name of the first useful alias it finds. None if
+                   none are found.
+
+        For example, "The Beatles" has alias "Le Double Blanc" with sort name
+        "Double Blanc, Le", so it's not considered. But it also has alias
+        "The Beatles" with sort name "Beatles, The", so this one is chosen.
+        Another example, "The Continuing Story of Bungalow Bill" has an alias
+        with the sort name equal to the title, this is not considered because
+        it makes more sense to swap the prefix.
+        """
+        for alias in aliases:
+            sortname = alias["sort-name"]
+            if (alias["name"].casefold() == name.casefold() and
+                    not sortname.casefold() == name.casefold()):
+                log.info("Enhanced Titles: sort name found for \"" + name + "\", \"" + sortname + "\".")
```

Comment:
`log.info()` can use % format strings (see https://github.com/metabrainz/picard/blob/90a8ee5a0d4d72d68a197195fdc9f181643fc6c8/picard/browser/browser.py#L121 for an example)

```suggestion
                log.info('Enhanced Titles: sort name found for "%s", "%s"', name, sortname)
```


That's much easier to read.